tests/testing_utils/fixtures.py

```diff
@@ -429,6 +430,11 @@ def native_app_extension_function(
     return NativeAppExtensionFunction(**native_app_extension_function_raw_data)
 
 
+@pytest.fixture
+def current_sanitized_username():
+    return get_sanitized_username()
```

Comment:
why? Can't we just call the function when we need the value?

Reply:
remove this whole thing. Having username (template default) accessed at class build time causes issues with integration tests as well. I've updated the PR to add fn.get_username() so this evaluated during templating. We don't need to access this in testing anymore.